### ISSUE TITLE
Visual Studio can't find an installed version from MSBuild

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#48063]
> The project file cannot be opened. Version 5.0.100-rc.1.20452.10 of the .NET Core SDK requires at least version 16.8.0 of MSBuild. The current available version of MSBuild is 16.7.0.37604. Change the .NET Core SDK specified in global.json to an older version that requires the MSBuild version currently available.

I hit this error in Visual Studio while I can't build from the command line without any problems.

I tried to run `dotnet msbuild -version` from the command line and got the following output:

```
Microsoft (R) Build Engine version 16.8.0-preview-20451-02+51a1071f8 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

16.8.0.45102
```

So MSBuild 16.8 already exists on my machine (but not VS 16.8). Why Visual Studio can't detect the existing MSBuild 16.8 and use it?

**Comment by RikkiGibson** [comment]
Using MSBuild 16.8 in Visual Studio pre-16.8 is not supported. Is that what you're trying to do?

**Comment by Youssef1313** [comment]
@RikkiGibson Yes exactly. I wanted to use an existing MSBuild 16.8 in VS 16.7.x

If it can't be supported for some reason, I'd suggest updating the error message.

The current message implies that MSBuild 16.8 isn't installed at all.
